File: personalization/android/access_session_data.java
// Returns the time elapsed in seconds
int timeElapsed = cleverTap.session.getTimeElapsed();

// Total App Opens
int totalVisits = cleverTap.session.getTotalVisits();

// Activity View Counts
int screenCount = cleverTap.session.getScreenCount();

// Last App Open
// If this is the user's first visit, the call below will return -1
Date lastVisit = new Date(cleverTap.session.getPreviousVisitTime() * 1000l);

File: personalization/ios/access_session_data.m
// Returns the time elapsed in seconds
int timeElapsed = [[CleverTap sharedInstance] sessionGetTimeElapsed];

// Total App Opens
int totalVisits = [[CleverTap sharedInstance] userGetTotalVisits];

// View Controller Counts
int screenCount = [[CleverTap sharedInstance] userGetScreenCount];

// Last App Open
// If this is the user's first visit, the call below will return -1
NSDate *lastTimeAppLaunched = [[NSDate alloc] initWithTimeIntervalSince1970:[[CleverTap sharedInstance] userGetPreviousVisitTime]];
